gwcs/wcstools.py

```diff
@@ -311,8 +311,9 @@ def fit_wcs_from_points(xy, world_coordinates, proj_point, projection=projection
     projection : `~astropy.modeling.projections.Projection` or str
         A projection type. One of the projections in `~astropy.modeling.projections.projcode`.
         Default is ``TAN`` projection.
-    degree : int
+    degree : int, None
         Degree of Polynpomial model to be fit to data.
+        An int degree is required for polynomials.
```

Comment:
@nden Please document what is the actual degree used when `distortion_type` is of "polynomial type" and `degree` is set to `None`.

Reply:
Oh, maybe I mis-understood: Is this a required parameter now? That is, if users choose "polynomial" type distortions they **have to** provide a degree, right?

Reply:
Yes, it's required now if distorion is polynomial.

Reply:
why not simply ignore `degree` in the future when (if ever) there will be support for non-polynomial distortions and user chooses such a non-polynomial distortion (just ignore `degree`)